### DISCUSSION TITLE
ScriptReference/Physics2D.Distance

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#635] [Notes]
# ScriptReference/Physics2D.Distance

https://docs.unity3d.com/ScriptReference/Physics2D.Distance.html

**Comment by qwe321qwe321qwe321** [comment]
[Physics2D.Distance returns wrong result with circle-like capsule inside the box collider.](https://forum.unity.com/threads/physics2d-distance-returns-wrong-result-with-circle-like-capsule-inside-the-box-collider.1150232/)
